File: hash.java
import java.util.HashMap;
import java.util.Map;

public class DrinksMap {
    public static void main(String[] args) {
        // создаём словарь (HashMap) где ключ - String, значение - Integer
        Map<String, Integer> hash = new HashMap<>();

        hash.put("beer", 100);
        hash.put("wisky", 650);
        hash.put("vodka", 350);

        // выводим содержимое
        for (Map.Entry<String, Integer> entry : hash.entrySet()) {
            System.out.println(entry.getKey() + ", " + entry.getValue());
        }
    }
}

File: hash.cpp
#include <iostream>
#include <map>
#include <string>

int main() {
    // создаём словарь (map) где ключ - string, значение - int
    std::map<std::string, int> hash;

    hash["beer"] = 100;
    hash["wisky"] = 650;
    hash["vodka"] = 350;

    // выводим содержимое
    for (const auto& pair : hash) {
        std::cout << pair.first << ", " << pair.second << std::endl;
    }

    return 0;
}
